lib/tasks/get-space-data.js

```diff
@@ -107,6 +107,29 @@ export default function getFullSourceSpace ({
       }),
       skip: () => skipContent
     },
+    {
+      title: 'Fetching tags data',
+      task: wrapTask((ctx, task) => {
+        return pagedGet({source: ctx.environment, method: 'getTags'})
+            .then(extractItems)
+            .then((items) => {
+              ctx.data.tags = items
+            })
+            .catch(e => {
+              console.log(`Caught error: ${e}`)
```

Comment:
Q: what kind of error do we expect here?

Reply:
mostly just 404 if the feature isn't enabled. I don't log it anymore. If the error is caught and handled then the export reports that it completed successfully 👍